R/drive_list.R

```diff
@@ -0,0 +1,162 @@
+#' List files on Google Drive
+#'
+#' @param path character vector, path name(s) where the Google drive files are
+#'   that you would like to list. Defaults to the "My Drive" directory.
+#' @param pattern character vector, regular expression(s) of title(s) of
+#'   documents to output in a tibble. If it is `NULL` (default), information
+#'   about all documents in drive will be output in a tibble.
```

Comment:
I agree with removing the last sentence. I *think* how we are going about it with `pattern` now makes sense in order to implement regular expressions (google drive lets one search using [contains, =, !=](https://developers.google.com/drive/v3/web/search-parameters), which probably serves most people's uses, but I could imagine some cases where our regex capability is quite useful.